website/docs/about/03--accessibility-statement.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Accessibility Statement
+description: This is an accessibility statement from the HashiCorp Design Systems team.
```

Comment:
This description doesn't feel like it's adding extra value, so we should consider removing it.